diesel_derives2/src/insertable.rs

```diff
@@ -0,0 +1,82 @@
+use syn;
+use quote;
+use proc_macro2::Span;
+
+use field::*;
+use model::*;
+use util::*;
+
+pub fn derive(item: syn::DeriveInput) -> Result<quote::Tokens, Diagnostic> {
+    let model = Model::from_item(&item)?;
+
+    if model.fields().is_empty() {
+        return Err(Span::call_site()
+            .error("Cannot derive Insertable for unit structs")
```

Comment:
I think the previous error was slightly better, however what about explicitly saying `Cannot derive Insertable for empty Structs`